### ISSUE TITLE
Current employees in PROD are missing email addresses

### ISSUE THREAD

**Issue by Travis-A-Clark** [bcgov/performance#911]
In PROD, there are approximately 100 employees that have records in the current employees list but that do not have an email listed. You can find them by sorting the employee list by email entry. Some of them have created goals or signed off on performance conversations in the past, so it seems they previously were able to authenticate into the system.

Can you please explore what might be causing this and what the impacts are? Thank you!

Potential that this could be relevant to the work going on in ticket #863 as well.

**Comment by telusdcinco** [comment]
Email address displayed on the Employee List page comes from the ODS employee demo data.  The 101 employees with blank emails have no email in the employee demo table.

![image.png](https://images.zenhubusercontent.com/60e88ef619e4760c8b3ee8bb/d08c6547-f1d5-4b7e-806a-94c612be32b2)

**Comment by telusdcinco** [comment]
As of this morning, 62 employees have blank email address on the Employee List.

![image.png](https://images.zenhubusercontent.com/60e88ef619e4760c8b3ee8bb/260b97b5-3314-4c1a-97b4-562d680dc2f2)

@Travis-A-Clark Should we change the source for the email address displayed on the Employee List or anywhere else, to use the stored email address in the User Profile, rather than the Employee Demo?

**Comment by Travis-A-Clark** [comment]
@telusdcinco No, let's leave it referencing the employee demo for now and work with Justin to try to figure out why the info is missing. If we need to change the source later we can. Thanks!
